Marlin/src/lcd/extui/lib/mks_ui/draw_wifi_list.cpp

```diff
@@ -93,6 +93,8 @@ void lv_draw_wifi_list(void) {
 
   lv_obj_t *buttonDown = lv_imgbtn_create(scr, "F:/bmp_pageDown.bin", OTHER_BTN_XPIEL * 3 + INTERVAL_V * 4, titleHeight + OTHER_BTN_YPIEL + INTERVAL_H, event_handler, ID_WL_DOWN);
   lv_obj_t *buttonBack = lv_imgbtn_create(scr, "F:/bmp_back.bin", OTHER_BTN_XPIEL * 3 + INTERVAL_V * 4, titleHeight + (OTHER_BTN_YPIEL + INTERVAL_H) * 2, event_handler, ID_WL_RETURN);
+  UNUSED(buttonDown);
+  UNUSED(buttonBack);
```

Comment:
If these are not used, why save them at all?

It does raise some concern that this might be allocating memory and leaking it. I don't know what lv_imgbtn_create does. It is is returning a pointer into a static pool, then we are fine and you could delete these variables.

If it is calling `new` and returning the result, then we have a problem.

Reply:
lvgl will free all children of a screen, when the is screen be freed. It's have a strong object hierarchy.